packages/admin/dashboard/src/routes/orders/order-create-fulfillment/components/order-create-fulfillment-form/order-create-fulfillment-form.tsx

```diff
@@ -29,27 +29,35 @@ export function OrderCreateFulfillmentForm({
 }: OrderCreateFulfillmentFormProps) {
   const { t } = useTranslation()
   const { handleSuccess } = useRouteModal()
+  const [searchParams] = useSearchParams()
+  const requiresShipping = searchParams.get("requires_shipping")
```

Comment:
nit(non-blocking): usually we would read this in the page container and send it as a boolean prop to the form

Reply:
Can you elaborate on this one? Page container as in the route container?

Reply:
yes, for this case in: `/dashboard/src/routes/orders/order-create-fulfillment/order-create-fulfillments.tsx`

Reply:
Should we attend to this before we merge?

Reply:
@riqwan, just pinging here in case you missed it.

Reply:
yes, missed this one! Will patch it up

Reply:
done! 🍻